secboot/secboot_tpm.go

```diff
@@ -185,6 +185,41 @@ func MeasureSnapModelWhenPossible(findModel func() (*asserts.Model, error)) erro
 	return nil
 }
 
+// LockTPMSealedKeys manually locks access to the sealed keys. Meant to be
+// called in place of passing lockKeysOnFinish as true to
+// UnlockVolumeUsingSealedKeyIfEncrypted for cases where we don't know if a
+// given call is the last one to unlock a volume like in degraded recover mode.
+func LockTPMSealedKeys() error {
```

Comment:
It'd be nice if we refactor `UnlockVolumeUsingSealedKeyIfEncrypted()` to use this call in a followup maybe?